Controller/TreeController.php

```diff
@@ -26,53 +26,31 @@ class TreeController extends Controller
     /**
      * @var string
      */
-    private $repositoryName;
-
-    /**
-     * @var string
-     */
     private $template = 'SonataDoctrinePHPCRAdminBundle:Tree:tree.html.twig';
 
     /**
-     * @var array
-     */
-    private $defaults;
-
-    /**
-     * @var bool
-     */
-    private $confirmMove = false;
-
-    /**
      * @var \PHPCR\SessionInterface
      */
     private $session;
+    /**
+     * @var array
+     */
+    private $treeConfiguration;
 
     /**
      * @param ManagerRegistry $manager
      * @param string          $sessionName
-     * @param string          $repositoryName
+     * @param array           $treeConfiguration
```

Comment:
then lets add a docblock here that this is the same structure as defined in the Configuration class, apart from move becoming a boolean.